google/resource_kms_key_ring_test.go

```diff
@@ -4,10 +4,11 @@ import (
 	"fmt"
 	"testing"
 
+	"log"
```

Comment:
Spacing again ... but maybe there's something I don't know about the `"log"` package?

Reply:
I think it's just gofmt being confused. It doesn't try to change it if I reorder them manually though, so done.